easybuild/easyconfigs/r/R/R-3.4.3-foss-2017b-X11-20171023.eb

```diff
@@ -0,0 +1,1695 @@
+name = 'R'
+version = '3.4.3'
+x11ver = '20171023'
+versionsuffix = '-X11-%s' % x11ver
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+checksums = ['7a3cb831de5b4151e1f890113ed207527b7d4b16df9ec6b35e0964170007f426']
+
+configopts = "--with-pic --enable-threads --enable-R-shlib"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and we're installing them anyway below
+configopts += " --with-recommended-packages=no"
```

Comment:
@Helios07 Please move these lines down below `osdependencies`, we changed that consistently in #5485.

Also include `easybuild_version = '3.5.0'` (+ comments, see existing R easyconfigs) to make sure that the updated `EB_R` easyblock is used, otherwise you may end up with a suboptimal R installation (not properly linked to BLAS/LAPACK).